src/runner/driver-test-runner.js

```diff
@@ -90,30 +90,40 @@ export class DriverTestRunner {
   /**
    * Used for v2 tests to ensure proper settings.
    *
-   * @param {string} settings - "browseMode" "focusMode" for NVDA, "pcCursor" "virtualCursor"
-   *                 for JAWS., "defaultMode" for others.
```

Comment:
I remove the reference to JAWS because there is no JAWS functionality in this function